### DISCUSSION TITLE
v5.0.0 branch - build ping-pong apps

### DISCUSSION THREAD

**Discussion by stingdau1206** [Lora-net/LoRaMac-node#1312] [Q&A]
radio.h: No such file or directory in-branch v5.0.0

**Comment by mluis1** [comment]
Please refer to https://github.com/Lora-net/LoRaMac-node/discussions/1239 for v5.0.0 branch development current status.

The ping-pong examples haven't yet been updated. That's why.

I have implemented the required updates. However I did not had the time to perform the required tests for ping-pong, tx-cw and rx-sensi examples.

If you are willing to provide feedback. Please find attached the required update patch file.
[v5.0.0-non-lorawan-examples-update.patch.txt](https://github.com/Lora-net/LoRaMac-node/files/8649732/v5.0.0-non-lorawan-examples-update.patch.txt)

I am moving this issue to the discussion tab as v5.0.0 branch is still a work in progress.

**Reply by mluis1** [reply]
@stingdau1206 Had you chance to give a try to the provided patch?

Would be nice to get your feedback.
